src/library/simplified/extension/doubleclick-carriercode-flightnumber--u18.js

```diff
@@ -0,0 +1,12 @@
+    b['u18'] = '';
```

Comment:
would be great to add an example block at the beginning of this extension so we can visualize the end results and input data